backend/src/middlewares/validateExpenseAuthority.js

```diff
@@ -0,0 +1,30 @@
+const { getRawFacilityContacts } = require('../components/facilities')
+const { getRawFundingAgreementById } = require('../components/fundingAgreements')
+const log = require('../components/logger')
+
+/**
+ * Validates that the user is an Expense Authority for the FA's facility.
+ * @param
+ * @returns
+ */
+module.exports = function () {
+  return async function (req, res, next) {
+    log.verbose(`validating expense authority`)
+
+    const contactId = req.session?.passport?.user?.contactId 
+    // Validate that the contactId represents the currentUser as you can only sign for yourself
+    if (req?.body.contactId !== contactId) {
+      res.sendStatus(403)
+      return
```

Comment:
Do we mean to explicitly return `undefined` here?  Why not `return res.sendStatus(403)`?